.github/workflows/code-health.yml

```diff
@@ -29,14 +29,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
it would be a good test to leave this as it is and see that dependabot catches correctly :-)

Reply:
there are many other actions to updates that this PR does not update 😨